lib/connectors_sdk/confluence_cloud/http_call_wrapper.rb

```diff
@@ -44,7 +44,7 @@ def custom_client_error
       end
 
       def config(params)
-        ConnectorsSdk::Atlassian::Config.new(:base_url => base_url(params[:cloud_id]), :cursors => params.fetch(:cursors, {}) || {})
+        ConnectorsSdk::Atlassian::Config.new(:base_url => "#{params[:external_connector_base_url]}/wiki", :cursors => params.fetch(:cursors, {}) || {})
```

Comment:
What if the provided base URL already contains `wiki`?

Reply:
That indicates the operator misconfigured the connector. It actually fails when trying to create the content source:

https://github.com/elastic/connectors/blob/9f25f35e17ffb36dfda754d657794ed9b5d2d75a/lib/connectors_sdk/confluence_cloud/authorization.rb#L58